vllm/engine/arg_utils.py

```diff
@@ -193,6 +194,16 @@ class EngineArgs:
     compilation_config: Optional[CompilationConfig] = None
     worker_cls: str = "auto"
 
+    # P/D disaggregation coonfiguration
+    kv_connector: Optional[str] = None
+    kv_buffer_size: Optional[float] = 1e9
+    kv_buffer_device: Optional[str] = "cuda"
```

Comment:
typo? there is one line below:

Recommended value: CPU.

Reply:
Fixed